src/components/UserDashboard/SavedQueries/index.js

```diff
@@ -191,10 +225,18 @@ export const MySavedQueries = compose(
                     ) : (
                       <Box mt={2} mb={2}>
                         <Scroll>
-                          {virtualStudies.map(s => (
-                            <Study>
+                          {virtualStudies.map((s, index) => (
+                            <Study key={s.id}>
                               <Row justifyContent="space-between" width="100%">
-                                <StudyLink to={`/explore?id=${s.id}`}>{s.name}</StudyLink>
+                                <div className={`${studyStyle}`}>
+                                  <StudyLink to={`/explore?id=${s.id}`}>{s.name}</StudyLink>
+                                  {virtualStudyLoad(s.id, index, props)}
```

Comment:
That cause a query for every virtual studies of the user at each render.

Please move to the appropriate lifecyle event.